src/react-aria-modal.js

```diff
@@ -200,7 +200,7 @@ class Modal extends React.Component {
 
     return React.createElement(FocusTrap,
       {
-        focusTrapOptions: {
+        focusTrapOptions: props.focusTrapOptions || {
           initialFocus: props.focusDialog
             ? `#${this.props.dialogId}`
             : props.initialFocus
```

Comment:
I'm a little concerned about this situation. A user could pass the `focusDialog` or `initialFocus` props *in combination with* `focusTrapOptions`, without realizing that their `focusTrapOptions` will nullify those other props. Do you think we could adjust the code so that `focusDialog` and `initialFocus` always make their way through to the `FocusTrap` component, even if `focusTrapOptions` is also provided?